implementation/test/TBTCSystemTest.js

```diff
@@ -69,55 +69,181 @@ describe("TBTCSystem", async function() {
     })
   })
 
-  describe("setSignerFeeDivisor", async () => {
-    it("sets the signer fee", async () => {
-      await tbtcSystem.setSignerFeeDivisor(new BN("201"))
+  describe("update Signer fee", async () => {
+    describe("beginSignerFeeDivisorUpdate", async () => {
+      const newFee = new BN("201")
+      it("executes and fires SignerFeeDivisorUpdateStarted event", async () => {
+        receipt = await tbtcSystem.beginSignerFeeDivisorUpdate(newFee)
 
-      const signerFeeDivisor = await tbtcSystem.getSignerFeeDivisor()
-      expect(signerFeeDivisor).to.eq.BN(new BN("201"))
+        expectEvent(receipt, "SignerFeeDivisorUpdateStarted", {
+          _signerFeeDivisor: newFee,
+        })
+      })
+
+      it("reverts if msg.sender != owner", async () => {
+        await expectRevert.unspecified(
+          tbtcSystem.beginSignerFeeDivisorUpdate(newFee, {
+            from: accounts[1],
+          }),
+          "",
+        )
+      })
+
+      it("reverts if fee divisor is smaller than 10", async () => {
+        await expectRevert(
+          tbtcSystem.beginSignerFeeDivisorUpdate(new BN("9")),
+          "Signer fee divisor must be greater than 9, for a signer fee that is <= 10%.",
+        )
+      })
     })
 
-    it("reverts if msg.sender != owner", async () => {
-      await expectRevert.unspecified(
-        tbtcSystem.setSignerFeeDivisor(new BN("201"), {
-          from: accounts[1],
-        }),
-        "",
-      )
+    describe("finalizeSignerFeeDivisorUpdate", async () => {
+      it("reverts if the governance timer has not elapsed", async () => {
```

Comment:
In this test case we verify the case where update was not initiated. Can we rename it and add a case where delay time has not elapsed (`await increaseTime(remainingTime)`)?